pyserini/search/lucene/irst/__main__.py

```diff
@@ -17,7 +17,7 @@
 import json
 import os
 import subprocess
-from pyserini.search.lucene.tprob import TranslationProbabilitySearcher
+from pyserini.search.lucene.irst import TranslationProbabilitySearcher
```

Comment:
Should we call it `LuceneIrstSearcher` to be more accurate?

Below, probably makes sense to call:
`pyserini/search/lucene/irst/_search_irst.py` -> `_searcher.py`? I think this makes it consistent with lucene package?